react-hooks/intro-to-react-hooks/src/exercises/exercise1/part1.js

```diff
@@ -22,8 +22,8 @@
 
   💡 We provide a "Field" component which reduces some complexity
   💡 in how much code is necessary. Take a look to see how 
-  💡 it functions, and what you will need to do to make its 
-  💡 value stateful. You do not need to modify the Field component.
+  💡 it functions, and what you will need to do to make it's 
```

Comment:
Same thing as above, `its` is the correct usage here because it indicates possessiveness.